examples/index.json

```diff
@@ -4234,7 +4234,7 @@
         "name": "Serialization/Deserialization",
         "url": "avro-serdes",
         "verifyBuild": true,
-        "verifyOutput": true,
+        "verifyOutput": false,
```

Comment:
What is the reason to disable this BBE verification?

Reply:
When building the BBE, the latest version of the Avro library is pulled from the central repository (as there is a new release) which generates this additional log message.
`ballerina/avro:1.0.0 [central.ballerina.io ->/home/runner/.ballerina/repositories/central.ballerina.io/bala/ballerina/avro/1.0.0]   17% [> ballerina/avro:1.0.0 [central.ballerina.io ->/home/runner/.ballerina/repositories/central.ballerina.io/bala/ballerina/avro/1.0.0]  100% [`.

This additional output causes a mismatch between the actual output of the Avro BBE and the expected result.

https://github.com/ballerina-platform/ballerina-distribution/actions/runs/9102412265/job/25021940100#step:4:562